.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,17 +1,48 @@
-**What does this PR do / Why do we need it**:
+**What this PR does / why we need it**:
 
-**Does this PR close any issues?**:<!-- optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged -->
+**Which issue(s) this PR fixes**:
+<!--optional, in `fixes #<issue number>` format, will close the issue(s) when PR gets merged-->
 Fixes #
 
+**What type of PR is this?**
+<!--
+Add one of the following kinds:
+/kind bug
+/kind cleanup
+/kind documentation
+/kind feature
+/kind design
+
+Optionally add one or more of the following kinds if applicable:
+/kind api-change
+/kind deprecation
+/kind failing-test
+/kind flake
+/kind regression
+/kind question
+-->
```

Comment:
This is useful if we want to switch to the upstream release-notes generation tool. We did it for KubeOne and I'm quite happy with how it works, so maybe we should do the same for other projects.

Reply:
these exact same /kind labels are good for everywhere?

Reply:
I expect some minor changes, e.g. that we'll add some more labels/kinds, but other than that, I think those should be good for everyone.